package.json

```diff
@@ -24,17 +24,18 @@
   },
   "homepage": "https://github.com/ember-learn/ember-help-wanted-webhook",
   "devDependencies": {
-    "mocha": "^2.4.5",
-    "mocha-only-detector": "^0.1.0",
-    "sinon": "^1.17.3"
+    "bluebird": "3.3.5",
+    "bunyan": "1.8.1",
+    "firebase": "^2.4.2",
+    "firebase-token-generator": "2.0.0",
+    "github-webhook-handler": "^0.5.0"
```

Comment:
These have to be dependencies not dev dependencies in order for openstack to launch (see my earlier commit)